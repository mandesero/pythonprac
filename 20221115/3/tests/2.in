alp = Alpha(c=10, z=2, a=42)
try:
    alp.qqq = 213
except Exception:
    print("...")

try:
    print(alp.c)
    print(alp.q)
except Exception:
    print("|||")
print(str(alp) * 4)
try:
    alq = AlphaQ(qwerty=[], z=2, a=42)
except Exception:
    print("///")
alq = AlphaQ(b=AlphaQ(a=12), z=2, a=42)
print(alq, alq.b)


