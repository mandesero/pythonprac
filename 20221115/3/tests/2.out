...
10
|||
a: 42, c: 10, z: 2a: 42, c: 10, z: 2a: 42, c: 10, z: 2a: 42, c: 10, z: 2
///
a: 42, b: a: 12, z: 2 a: 12
